libmariadb/ma_errmsg.c

```diff
@@ -28,20 +28,20 @@ const char *SQLSTATE_UNKNOWN= "HY000";
 #ifdef GERMAN
```

Comment:
please remove entire #ifdef GERMAN block - it is a relict from libmysql 3.23.48 and not in use anymore